datafusion/core/src/datasource/physical_plan/parquet/row_groups.rs

```diff
@@ -416,11 +417,11 @@ mod tests {
     fn row_group_pruning_predicate_simple_expr() {
```

Comment:
Since I updated the signature of `prune_row_groups_by_statistics` I also had to update the tests appropriately